doc/source/ReleaseNotes/generate.py

```diff
@@ -1,8 +1,8 @@
 import re
 
 index = open( "./index.md", "w" )
-index.write( "Release Notes\n" )
-index.write( "=============\n\n" )
+index.write( "# Release Notes #\n\n" )
+index.write( "## Contents ##\n\n" )
```

Comment:
Is it necessary for this to be visible in the final html? It seems meaningless for the reader, its just a way of marking this page to be included in the TOC, right? Can we not do that with some kind of metadata that doesn't actually render?

Similar to what you did for the static pages for example:
https://github.com/GafferHQ/gaffer/pull/2616/commits/a6f1969ef444689205b9fa7184d68d16d63ffc8f#diff-028a89c5fff7229f3a20957bb8f5c381R10

Reply:
I should have done more research – yes, this can be accomplished using reST globbing. Derp. Time to rebase...